deploy/ansible/roles/nfs-server-setup/tasks/configure.yml

```
@@ -0,0 +1,41 @@
+---
+
+- name: Ensure the NFS utils package is installed
+  package:
+    name: nfs-utils
+    state: present
+
+- name: Ensure NFS Server is running
+  systemd:
+    name: nfsserver
+    state: started
+    enabled: yes
+
+- name: Ensure expected base directories are present
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: +r
+  with_items:
+    - "{{ target_media_location }}"
+    - "/usr/sap/{{ app_sid | upper }}/SYS"
+    - "/tmp/app_template"
+    - "/sapmnt/{{ app_sid | upper }}/global"
+    - "/sapmnt/{{ app_sid | upper }}/profile"
+
+- name: Ensure exports file exists
+  template:
+    src: exports.j2
+    dest: /etc/exports
+    owner: root
+    group: root
+    mode: 0644
```

While given the context there are probably no other NFS exports defined in this file at this time, I'd prefer to see a "no-potential-clobber" approach using `lineinfile` instead of a template

```suggestion
- name: Ensure exports file exists
  lineinfile:
    dest: /etc/exports
    create: yes
    line: "{{ item.path }}    {{ item.export_settings }}"
    regexp: "^{{ item.path }}\s+"
    owner: root
    group: root
    mode: 0644
    state: present
```